Add /json endpoint

astericky opened the pull request:
add /json endpoint

astericky committed:
add /json route and update data class

pnikonowicz reviewed (commented):
functionally i think everything is fine, but i think the body of this change should be constrained to main.py and the `handleJSON` function. 

Or, we can just take everything as is, but i felt it important to at least have the discussion about the various other changes.

pnikonowicz commented:
there are some commits that contradict each other, can we inline those to make the git history simpler?

astericky commented on common/data.py:
```diff
@@ -17,8 +17,10 @@ class Data:
-        data['similar_events'] = [event.to_dict() for event in self.similar_events]
+
+        # Ensure similar_events is recursively convertedclea
+        if not self.similar_events:
```
this breaks if I don't have the condition showing there is at least one value in similar_events property

astericky committed:
update data class

pnikonowicz committed:
cleanup: remove cosmetic

these changes appear to be unnecessary and / or
unintentional.

if these changes are needed. please open a new PR
